content/04.baselines.md

```diff
@@ -1,14 +1,16 @@
 ## Tip 2: Use traditional methods to establish performance baselines {#baselines}
 
-It is important to implement a simple model to establish an adequate performance baseline, since it is easy to fall into the trap of an unnecessarily convoluted analysis.
-For example, a researcher could first a build a logistic regression model using the same software framework planned for DL to first evaluate the classification performance.
-This approach will help researchers with assessing the complexity of the task at hand and debugging more complex DL architectures.
-The utility of these methods is evidenced by the recent development of hybrid models which combine DL and simpler models to improve robustness, interpretability, and confidence estimation [@arxiv:1803.04765; @arxiv:1805.11783].
-Depending on the amount of available data and the type of tasks, DL models may not necessarily perform the best.
-As an illustration, the simple baseline models by Rajkomar et al. [@doi:10.1038/s41746-018-0029-1] achieved performance comparable with that of DL in a number of clinical prediction tasks using electronic health records, which may be a surprise to many.
+Since deep learning requires practitioners to consider a larger number and variety of tuning parameters compared to more traditional methods, it is easy to fall into the trap of performing an unnecessarily convoluted analysis. 
+Hence, before applying deep learning to a given problem, we highly recommend implementing a simple model at the beginning of each study to establish adequate performance baselines.
```

Comment:
I totally agree with this point. It might be better worded a bit differently, since implementation is not always necessary (e.g. sometimes there is an existing baseline to compare to), but there is value in familiarizing ones self with a software framework before applying it to the problem at hand. It may also be worth mentioning encouraging some exploratory data analysis before they even implement/identify a baseline model.

Reply:
Hm yeah, but I think we do not necessarily want to rely on baselines from literature alone. Imho, we should add your point though, saying that also baselines available from literature should be considered. Reasons for implementing the simple method here are generally

- not all literature results are trustrworthy ;)
- baselines may not be available
- implementations can be complicated
- if a complicated neural net is not training, it may not be clear whether it's due to the settings, implementation errors, or simply weird things happening with the dataset (issues with data prep, etc.)